dev/fattest.simplicity/src/componenttest/rules/repeater/RepeatTests.java

```diff
@@ -111,5 +111,14 @@ public void evaluate() throws Throwable {
                 }
             }
         }
+
+        private static boolean shouldRun(RepeatTestAction action) {
+            String repeatOnly = System.getProperty("fat.repeat.only");
+            if (repeatOnly == null) {
```

Comment:
if `repeatOnly == null`, shouldn't we be returning `action.isEnabled()` instead of `true` all the time? (existing behavior before this sys prop was introduced)

Reply:
Switched this around